pkgs/development/python-modules/txtai/default.nix

```diff
@@ -0,0 +1,146 @@
+{
+  lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pythonRelaxDepsHook
+# propagated build input
+, faiss
+, torch
+, transformers
+, huggingface-hub
+, numpy
+, pyyaml
+, regex
+# optional-dependencies
+, aiohttp
+, fastapi
+, uvicorn
+# , apache-libcloud
```

Comment:
```suggestion
```

One TODO about a missing optional dependency is probably sufficient..

Reply:
I would like to keep the order if possible, it makes it easier for someone to parse the setup.py file and understand what is missing and what has changed.